packages/fxa-auth-server/config/rate-limit-rules.txt

```diff
@@ -1,145 +1,105 @@
-  sendUnblockCode                       : ip                : 5             : 10 minutes        : 30 minutes  : block
-  unblockEmail                          : email             : 5             : 15 minutes        : 15 minutes  : block
+  createEmail                           : ip_uid            : 5           : 15 minutes      : 15 minutes  : block
+  recoveryEmailResendCode               : ip_uid            : 5           : 15 minutes      : 15 minutes  : block
+  recoveryEmailSecondaryResendCode      : ip_email          : 5           : 15 minutes      : 15 minutes  : block
+  recoveryEmailSecondaryResendCode      : ip                : 100         : 24 hours        : 24 hours    : block
+  passwordForgotSendCode                : ip_email          : 5           : 15 minutes      : 15 minutes  : block
+  passwordForgotSendCode                : ip                : 100         : 24 hours        : 24 hours    : block
+  passwordForgotResendCode              : ip_email          : 5           : 15 minutes      : 15 minutes  : block
+  passwordForgotResendCode              : ip                : 100         : 24 hours        : 24 hours    : block
+  sendVerifyCode                        : ip_uid            : 5           : 15 minutes      : 15 minutes  : block
+  sendVerifyCode                        : ip                : 100         : 24 hours        : 24 hours    : block
+  sendUnblockCode                       : ip_email          : 5           : 15 minutes      : 15 minutes  : block
+  sendUnblockCode                       : ip                : 100         : 24 hours        : 24 hours    : block
+  unblockEmail                          : ip_email          : 5           : 15 minutes      : 15 minutes  : block
 
 # Bad Login Attempts - These are specific to login failures. Again, the user hasn't authenticated yet, so we check both
 # email and ip.
-  accountLogin                          : email             : 5             : 15 minutes        : 15 minutes  : block
-  accountLogin                          : ip                : 100           : 24 hours          : 24 hours    : block
-  passwordChange                        : email             : 5             : 15 minutes        : 15 minutes  : block
-  passwordChange                        : ip                : 100           : 24 hours          : 24 hours    : block
-  authenticatedAccountLogin             : uid               : 5             : 15 minutes        : 15 minutes  : block
-  authenticatedPasswordChange           : uid               : 5             : 15 minutes        : 15 minutes  : block
+  accountLogin                          : ip_email          : 5           : 15 minutes      : 15 minutes  : block
+  accountLogin                          : ip                : 100         : 24 hours        : 24 hours    : block
+  authenticatedAccountLogin             : ip_uid            : 5           : 15 minutes      : 15 minutes  : block
```

Comment:
Side note looking through these, is there a way to have V2 customs in a "report only" mode? Some of these limits seem low and would be nice to know how many folks would run into the block.

Reply:
There is not. I like the idea. If you think this is important feature, we can do a follow up to support it.